easybuild/easyconfigs/l/LinBox/LinBox-1.4.0-foss-2016a.eb

```diff
@@ -0,0 +1,39 @@
+##
+# This file is an EasyBuild recipe; see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright (c) 2016 Riccardo Murri <riccardo.murri@gmail.com>
+# Authors::   Riccardo Murri <riccardo.murri@gmail.com>
+# License::   GPL
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'LinBox'
+version = '1.4.0'
+
+homepage = 'http://linalg.org/'
+description = "C++ library for exact, high-performance linear algebra"
+
+toolchain = {'version': '2016a', 'name': 'foss'}
+
+sources = ['v%(version)s.zip']
+source_urls = ['https://github.com/linbox-team/linbox/archive']
+
+builddependencies = [
+    ('Autotools', '20150215'),
+]
+dependencies = [
+    ('FFLAS-FFPACK', '2.2.0'),
+    ('Givaro',       '4.0.1'),
```

Comment:
we don't do this aligning anywhere else, and our automagic style checker will trip over this...